### PR TITLE
Add generics example

### PR DESCRIPTION
**Pull request by jayvdb**
I expect this may be able to be improved, in which case I will learn a bit along the way.

### CODE REVIEW

File: examples/generics-actix/src/main.rs [hunk 1]
```diff
@@ -0,0 +1,117 @@
+use std::{error::Error, net::Ipv4Addr};
+
+use actix_web::{
+    get, middleware::Logger, web::Json, web::Query, App, HttpServer, Responder, Result,
+};
+use serde::{Deserialize, Serialize};
+use utoipa::{
+    openapi::{schema::Object, schema::ObjectBuilder},
```
Comment (jayvdb) [inline examples/generics-actix/src/main.rs:8]: can merge these two

File: examples/generics-actix/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,117 @@
+use std::{error::Error, net::Ipv4Addr};
+
+use actix_web::{
+    get, middleware::Logger, web::Json, web::Query, App, HttpServer, Responder, Result,
```
Comment (jayvdb) [inline examples/generics-actix/src/main.rs:4]: and merge the two `web::`

File: examples/generics-actix/src/main.rs (hunk 1 above)
Comment (juhaku) [reply examples/generics-actix/src/main.rs:8]: Do yo mean, that the `web::Json` and `web::Query` could be provided via `utoipa` as `utoipa::Json` and `utoipa::Query`?

Comment (jayvdb) [reply examples/generics-actix/src/main.rs]: no. I was just noting the `use` syntax was not optimal.

Comment (juhaku) [reply examples/generics-actix/src/main.rs]: Ahh, you meant the imports. Sure they can.

**Pull request merged**